.github/workflows/_deploy.yml

```
@@ -236,8 +236,6 @@ jobs:
         working-directory: mdn/fred
         run: |
           npm run ssr
```

Nit: Unnecessary multiline

```suggestion
        run: npm run ssr
```